examples/with-apollo/lib/initApollo.js

```diff
@@ -0,0 +1,37 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
```

Comment:
They can be independent and still share some code, no? Anyway just a personal opinion (i.e. I wouldn't do this in my repo).